scripts/zones/Yuhtunga_Jungle/IDs.lua

```diff
@@ -63,6 +65,7 @@ zones[tpz.zone.YUHTUNGA_JUNGLE] =
         OVERSEER_BASE         = 17281600,
         CERMET_HEADSTONE      = 17281625,
         PEDDLESTOX            = 17281640,
+        TUNING_OUT_QM         = 17281590,
```

Comment:
Oh, on closer examination of the ID file, could you move this `17281590` definition up a bit to be in numerical order after the `BLUE_RAFFLESIA_OFFSET = 17281586,` offset?